src/developer_config.yaml

```diff
@@ -135,7 +137,7 @@ export_choices:
   export_intram_by_sic: "output_intram_by_sic_2023-12-06_v68.csv" # Can be set to None
   export_status_filtered: None # Can be set to None
   export_fte_total_qa: "total_fte_qa_2023-12-06_v68.csv" # Can be set to None
-  export_frozen_long_form: None # Can be set to None
+  export_long_form: None # Can be set to None
```

Comment:
Tiny thing, but just for consistency, take out the underscore between long and form?